src/MF_Modules/MFAnalog.cpp

```diff
@@ -17,16 +17,35 @@ MFAnalog::MFAnalog(uint8_t pin, analogEvent callback, const char * name, uint8_t
 }
 
 void MFAnalog::update()
-{    
+{
+/* *****************************************************************************
+*   There is a pull request to transfer the millis() to the mobiflight.cpp
+*   this will save some memory and would be also valid for tick();
+********************************************************************************/
     uint32_t now = millis();
+    if (now-_lastTick > 10) {
+      tick();
+      _lastTick = millis();
+    }
     if (now-_last <= 50) return; // Analog is too spammy on the protocol to MF otherwise.
+/* *************************************************************************** */
 
-    int newValue = (int) analogRead(_pin);
+    int newValue = ADC_Average_Buffer/ADC_MAX_AVERAGE;
     _last = now;
+//Serial.print("Analog: "); Serial.println(newValue);
```

Comment:
I assume this was just for testing purposes and should be removed from the final checked-in code?

Reply:
Yes, deleted it.